### PR TITLE
Clippy

### PR DESCRIPTION
**Pull request by kinggoesgaming**
image

~~This is declared a breaking change as per clippy requesting that `Hash` be derived, as @Dylan-DPC asked.~~

### CODE REVIEW

**Comment by KodrAus** [comment]
@kinggoesgaming Deriving `Hash` shouldn't be a breaking change, especially since it was already implemented before.

File: src/lib.rs [hunk 1]
```diff
@@ -995,11 +997,11 @@ impl fmt::LowerHex for Uuid {
     }
 }
 
-impl hash::Hash for Uuid {
-    fn hash<S: hash::Hasher>(&self, state: &mut S) {
-        self.bytes.hash(state)
-    }
-}
+//impl hash::Hash for Uuid {
```
Comment (KodrAus) [inline src/lib.rs]: We can probably just remove this commented out code.

Comment (kinggoesgaming) [reply src/lib.rs]: oops

**Comment by kinggoesgaming** [comment]
Right I will re-target it to `master`

**Comment by Dylan-DPC** [comment]
bors: r+

**Comment by Dylan-DPC** [comment]
bors: retry

**Comment by Dylan-DPC** [comment]
bors: try

**Comment by Dylan-DPC** [comment]
bors: try

**Comment by Dylan-DPC** [comment]
bors: r+

**Pull request merged**